style.css

```diff
@@ -0,0 +1,94 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: url(images/header-illsutration-mobile.png);
+  background-color: white;
```

Comment:
- [ ] As requested by the previous reviewer, since every section in the project has its own background image, and this one is specified by `Figma` only to the `.hero-section`, kindly move it from the body to the mentioned class.